mne/annotations.py

```diff
@@ -608,4 +617,20 @@ def events_from_annotations(raw, event_id=None, regexp=None,
     logger.info('Used Annotations descriptions: %s' %
                 (list(event_id_.keys()),))
 
+    if len(dropped) > 0:
+        dropped = list(set(dropped))
```

Comment:
we could give a more informative message, using a counter to state the types dropped and how many of each.